lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -52,9 +52,9 @@ module.exports = function(
    * We expose the whole service instead of the `.drives`
    * property, which is the one we're interested in since
    * this allows the property to be automatically updated
-   * when `DrivesModel` detects a change in the drives.
+   * when `availableDrives` detects a change in the drives.
```

Comment:
...and yet here we're implying that availableDrives is itself responsible for detecting drives being added / removed?